spec/support/create_strategy_for_repository_pattern.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class CreateStategyForRepositoryPattern
+  def association(runner)
+    runner.run
+  end
+
+  def result(evaluation)
```

Comment:
Okay, so it's just a generic repository thing, not something Valkyrie-specific (where these stages are defined as a part of the repositories as they exist here).